frontend/src/pages/projects/screens/detail/notebooks/NotebookTableRow.tsx

```diff
@@ -51,7 +42,7 @@ const NotebookTableRow: React.FC<NotebookTableRowProps> = ({
   const { currentProject } = React.useContext(ProjectDetailsContext);
   const navigate = useNavigate();
   const [isExpanded, setExpanded] = React.useState(false);
-  const { size: notebookSize, error: sizeError } = useNotebookDeploymentSize(obj.notebook);
+  const { selectedSize: notebookSize } = usePreferredNotebookSize();
```

Comment:
I don't know if `usePreferredNotebookSize` is giving you the right value here -- it doesn't know anything about the current row's notebook, it just gives the `currentUserState.lastSelectedSize`. I created a workbench with container size "Small", then I created one with size "Large", and they both display as "Small" with this change but they are correct on main.

Reply:
Yes, I saw the same too an hour or two ago .. there are a few things I am testing .. will update this shortly